flw/src/main/java/org/motechproject/nms/flw/controller/FrontLineWorkerUpdateLanguageController.java

```diff
@@ -0,0 +1,62 @@
+package org.motechproject.nms.flw.controller;
+
+import org.motechproject.alerts.contract.AlertService;
+import org.motechproject.alerts.domain.AlertStatus;
+import org.motechproject.alerts.domain.AlertType;
+import org.motechproject.nms.csv.exception.CsvImportException;
+import org.motechproject.nms.flw.service.FrontLineWorkerUpdateLanguageImportService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+@Controller
+public class FrontLineWorkerUpdateLanguageController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FrontLineWorkerUpdateLanguageController.class);
+
+    private AlertService alertService;
+
+    private FrontLineWorkerUpdateLanguageImportService flwUpdateLanguageImportService;
+
+    @RequestMapping(value = "/updateLanguage", method = RequestMethod.POST)
```

Comment:
Do we really need a separate controller for updates or can we merge this file with FrontLineWorkerImportController?